packages/core/src/utils/resolveClusterClick.ts

```diff
@@ -0,0 +1,16 @@
+import { Feature, Map } from 'ol'
+import { createEmpty, extend } from 'ol/extent'
+
+export const resolveClusterClick = (map: Map, feature: Feature) => {
+  const features = feature.get('features')
```

Comment:
`feature.get('features')` resolves to any, thus the iteration with `forEach` is typed with `any` as well.

Please add an appropriate type

Reply:
https://github.com/Dataport/polar/pull/42/commits/0e33f6c9d4f21e5e8e8e8fa62d8e48ed4b78a908